src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -417,7 +417,7 @@ private JObject GetMinimalPayload(string hostId, JArray triggersArray)
                 { "triggers", triggersArray }
             };
 
-            if (_environment.IsFlexConsumptionSku())
+            if (_environment.IsFlexConsumptionSku() || _environment.IsConsumptionV1RunningOnLegion())
```

Comment:
Why are you making this change? We don't intend to change host behaviors for Cv1 when running on Legion. It's just a migration.

Reply:
The reason for the change is because our current IsFlexConsumptionSku() doesn't actually check the sku of the pods correctly. It will return true for both flex consumption and migrated cv1 pods as long as it's running on legion. And if we want any variance between the migrated cv1 and flex consumption, we will need to have a definitive way to distinguish between the two.

So far, there doesn't seem to be any changes needed for migrated cv1 apps for the host runtime for EUAP, but there could be a few. For example, we currently emit FES for flex consumption, but the mesh process doesn't do anything with the data, so we could get rid of it for flex consumption containers. However, I believe our plan is to maintain FES for migrated cv1 pods. Another change we could have for migrated cv1 and flex would be billing. I believe we want to keep the billing the same for migrated cv1 as the existing cv1.

@balag0 What do you think?